src/server/journal/serializer.cc

```diff
@@ -0,0 +1,227 @@
+// Copyright 2022, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#include "server/journal/serializer.h"
+
+#include "base/io_buf.h"
+#include "base/logging.h"
+#include "io/io.h"
+#include "server/common.h"
+#include "server/error.h"
+#include "server/journal/types.h"
+#include "server/main_service.h"
+#include "server/transaction.h"
+
+using namespace std;
+
+using nonstd::make_unexpected;
+
+// TODO: Stolen from rdb, unite in common utils
+
```

Comment:
See TODO:
I would even move all of them to some common folder

Reply:
rdb_load rdb_save rdb_extensions 
serializer -> rename to journal_serializer or split into journal_load and journal_write (and execute)

Reply:
But It'd be better to do this afterwards because it'll be too many changes for one PR